lib/rules/no-extend-native.js

```diff
@@ -1,6 +1,7 @@
 /**
  * @fileoverview Rule to flag adding properties to native object's prototypes.
  * @author David Nelson
+ * @deprecated in ESLint v3.2.0
```

Comment:
Sorry, I don't remember what did we decide. Do we just leave the old rule unchanged, or should we remove all of the code and just do `module.exports = require('./no-global-assign');`? I guess former, since we don't want new changes to the rule to be propagated to deprecated version.